react-common/components/language/LanguageSelector.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+import { Link } from "../controls/Link";
+import { Modal } from "../controls/Modal";
+import { LanguageCard } from "./LanguageCard";
+
+
+const defaultLanguages = ["en"];
+
+interface LanguageSelectorProps {
+    visible: boolean;
+    activityOpen: boolean;
```

Comment:
Do you need this `activityOpen` prop? I don't see it being used.

Reply:
Ah good point, leftover from my various experiments. I'll remove it.